chrome/content/zotero/xpcom/reader.js

```diff
@@ -548,7 +550,12 @@ class ReaderInstance {
 				}
 			},
 			onSaveCustomThemes: (customThemes) => {
-				Zotero.Prefs.set('reader.customThemes', JSON.stringify(customThemes));
+				if (customThemes?.length) {
+					Zotero.SyncedSettings.set(Zotero.Libraries.userLibraryID, 'readerCustomThemes', customThemes);
+				}
+				else {
+					Zotero.SyncedSettings.clear(Zotero.Libraries.userLibraryID, 'readerCustomThemes');
+				}
```

Comment:
`set` and `clear` are async, so we should technically make this function async both here and in the reader. Should at least do it here for now.

Reply:
Sure thing, converted here for now.